functions/export-to-s3/index.py

```diff
@@ -33,12 +33,14 @@ def handler(event, context):
     dbName = os.environ['DB_NAME'].replace(' ', '').split(',')
 
     if eventId in rdsEventID:
+        try:
+          snapshotDbName=boto3.client("rds").describe_db_snapshots(DBSnapshotIdentifier=sourceId)['DBSnapshots'][0]['DBInstanceIdentifier']
+        except: 
+          snapshotDbName=boto3.client("rds").describe_db_cluster_snapshots(DBClusterSnapshotIdentifier=sourceId)['DBClusterSnapshots'][0]['DBClusterIdentifier']
+
         for db in dbName:
-            matchSnapshotRegEx = "^rds:" + db + "-\d{4}-\d{2}-\d{2}-\d{2}-\d{2}$"
-            if re.match(matchSnapshotRegEx, sourceId):
```

Comment:
I have ended up doing a fork so that the regex can be provided in the dbName variable. This allows more flexibility on which db snapshots will fire an export. More specifically it allows the schedule for export to be dictated by the regex not the event so if you have a very large db you can have the snapshot daily but only export to S3 weekly or monthly for example.

Reply:
@CallumD thanks for your detailed comment on how you approached the adjustment of the module. We'll keep this into consideration.

Reply:
lgtm

Reply:
@awasilyev after testing this PR I validated that removing this regex the export  task Identifier ends up with `-` and this causes the task to fail at start.